api/openapi.yaml

```diff
@@ -440,6 +441,14 @@ components:
           x-oapi-codegen-extra-tags:
             rlp: "-"
             json: "yield,omitempty"
+        registrationBlock:
```

Comment:
Its better to reuse some other type. For example that one for a date, but rename it to optionalUInt64(nillableUInt64?) or something like that